docs/how-to-configure-new-repository.md

```diff
@@ -63,7 +63,10 @@ policies:
 
 ## Best practices
 
-It is recommended to set the following settings:
+It is recommended to follow these best practices:
 
 1. Only set "Allow squash merging" for the Merge button. It will ensure clean
    history for the repository.
+2. Set up a security scanning tool like, for example, [Github CodeQL](https://docs.github.com/en/code-security/secure-coding/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning).
+3. Set up a test coverage scanning tool like, for example, [Codecov](https://codecov.io/).
```

Comment:
anything needs to be said about the recent security issue with codecov? What's the best practices now?

Reply:
I added it as an example based on the fact that most of our repos already use it: https://codecov.io/gh/open-telemetry. It would be odd to advise against using it while it's already being used heavily. If you think we should reconsider it and look for alternatives, we can leave it out as an example for now and tackle this separately.